pkg/networkmanager/networking.scss

```diff
@@ -415,32 +415,16 @@ h2 + .onoff-ct {
 }
 
 #add-zone-dialog legend {
-    all: unset;
     color: var(--color-subtle-copy);
-    display: block;
-    font: inherit;
-    /* Align with label padding */
-    line-height: 1.5;
-    padding: 0 0.5rem;
     font-size: var(--pf-global--FontSize--sm);
 }
 
-#add-zone-dialog .add-zone-zones {
-    display: flex;
-    flex-flow: row wrap;
-    align-items: last baseline;
-}
-
 .add-zone-zones legend {
-    line-height: 3 !important;
+    line-height: 3;
 }
 
-#add-zone-dialog .add-zone-zones label {
-    margin: 0;
-    flex-flow: column;
-    padding: 0 0.5rem;
+#add-zone-dialog .add-zone-zones .pf-c-radio__label {
     text-transform: capitalize;
-    font-size: var(--pf-global--FontSize--sm);
```

Comment:
I'm wondering if we could get rid of even more CSS in the file.

Reply:
The more you can remove the better ;)

Reply:
I figure we'll do that in #15054, after this lands. I don't want to accidentally break stuff on a PR we're both happy with right now, on both of our last days this year. We can finish up the cleanup in 2021. 😉

There are blocks I'm 99% certain we can just remove (when the time comes).